cmd/agent/app/httpserver/collector_proxy.go

```diff
@@ -54,11 +64,24 @@ func (c *collectorProxy) GetSamplingStrategy(serviceName string) (*sampling.Samp
 	ctx, cancel := tchannel.NewContextBuilder(time.Second).DisableTracing().Build()
 	defer cancel()
 
-	resp, err := c.client.GetSamplingStrategy(ctx, serviceName)
+	resp, err := c.samplingClient.GetSamplingStrategy(ctx, serviceName)
```

Comment:
TODO: enable tracer on the tchannel and get metrics for free (sampler could be off)